### PR TITLE
adding a separate PIN show field, with a reset switch

### PR DESCRIPTION
**Pull request by rossatbond**
This PR adds a "show" like field in the BondCards object that can revel PINs if we know the route/method to call at our backend.

### CODE REVIEW

File: src/bond-sdk-cards.js [hunk 1]
```diff
@@ -84,6 +84,56 @@ class BondCards {
...
+    identity,
+    authorization,
+    reset = false,
+    htmlWrapper = "text",
+    htmlSelector,
+    css = {},
+  }) {
+
+    const requestParams = {
+      method: reset ? "PATCH" : "GET",
```
Comment (rossatbond) [inline src/bond-sdk-cards.js:113]: @kirbpowell coupled with the fincore PR

**Review by prithvi-bond — APPROVED** [review]

**Review by dominicsardina — APPROVED** [review]

File: src/bond-sdk-cards.js (hunk 1 above)
Comment (kirbpowell) [reply src/bond-sdk-cards.js:113]: [288](https://github.com/bond-tech/baas_financial_core/pull/288) or [290](https://github.com/bond-tech/baas_financial_core/pull/290)

Comment (rossatbond) [reply src/bond-sdk-cards.js:113]: @kirbpowell nominally 288 but now both

**Comment by rossatbond** [comment]
@prithvi-bond @dominicsardina just waiting for convergence on the `financial-core` things this will depend on

**Pull request merged**